packages/example-app-base/src/components/ComponentPage/ComponentPage.tsx

```diff
@@ -301,7 +301,7 @@ export class ComponentPage extends React.Component<IComponentPageProps, {}> {
 
     if (this.props.dos && this.props.donts) {
       dosAndDonts.push(
-        <div className="ComponentPage-doSections" key="do-sections">
+        <div className="ComponentPage-doSections" key="do-sections" id={!this.props.bestPractices ? 'BestPractices' : undefined}>
```

Comment:
You might consider using a local variable to keep `'BestPractices'` in-sync given how brittle this is (unrelated to your changes). But that change isn't necessary, just a thought.